agent/ecs_client/model/ecs/api.go

```diff
@@ -9385,9 +9386,6 @@ func (s ManagedAgentStateChange) GoString() string {
 // Validate inspects the fields of the type to determine if they are valid.
 func (s *ManagedAgentStateChange) Validate() error {
 	invalidParams := request.ErrInvalidParams{Context: "ManagedAgentStateChange"}
-	if s.Name == nil {
```

Comment:
is this check not required anymore?

Reply:
this code is autogenerated from the agent/ecs_client/model/api/api-2.json change using `make gogenerate`